src/client/pages/ConnectPayment/index.tsx

```diff
@@ -20,7 +23,11 @@ export const ConnectPayment: React.FC = () => {
           <title>{textKeys.ONBOARDING_CONNECT_DD_PAGE_TITLE()}</title>
         </Helmet>
         <TopBar>
-          <LanguagePicker path="/new-member/connect-payment" />
+          {market === Market.Se ? (
```

Comment:
Let's use a feature flag here - just let me know and I can introduce you to them! 

We want to avoid "market" checks in the code -- everything should be a global feature :)

Reply:
Ok! I would love some insights into how we use feature flags!

Reply:
Cool, just let me know and I can introduce you

Reply:
Oh, or just check if `currentLocale` (which you import from `useCurrentLocale` from `/client/l10n/useCurrentLocale.ts`) has the phone number stuff?

Reply:
That's one of the points of adding it to the `locales` object in my mind 😊 But then if we want to merge this we also need a feature flag to hide this until we know the correct opening hours etc.

@robinandeer @amandaflod

Reply:
Yes! Now Sonny wants some A/B testing as well so we will not be able to merge this without a feature flag before things are more specified! :)

Reply:
Now we're not doing A/B testing anymore! :P